### PR TITLE
Manually publish Nessie Helm chart to https://charts.projectnessie.org 

### PR DESCRIPTION
**Pull request by nastra**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/1720)

I ran into some issues with the `chart-releaser-action` where it
would try and only release charts that changed (helm/chart-releaser-action#71).
I tried adding some changes locally to the chart and then running the
same commands as the action does, but I couldn't get it to work
properly, therefore I decided to do the publishing manually via commit

### CODE REVIEW

File: .github/workflows/release-publish.yml [hunk 1]
```diff
@@ -251,6 +251,7 @@ jobs:
       if: ${{ !env.ACT }}
       with:
         charts_dir: helm
+        charts_url: https://projectnessie.org/charts/
```
Comment (snazy) [inline .github/workflows/release-publish.yml:254]: Update the URLs in the release-notes as well?

Comment (laurentgo) [reply .github/workflows/release-publish.yml:254]: But I can have charts.projectnessie.org point to it, I guess?

Comment (nastra) [reply .github/workflows/release-publish.yml]: > But I can have charts.projectnessie.org point to it, I guess?
yep, I changed the URL here to also point to charts.projectnessie.org

**Review by rymurr — DISMISSED** [review]

**Review by rymurr — APPROVED** [review]

File: .github/workflows/release-publish.yml [hunk 2]
```diff
@@ -231,13 +231,18 @@ jobs:
         mv nessie-${RELEASE_VERSION}.tgz nessie-helm-${RELEASE_VERSION}.tgz
         echo ::set-output name=NESSIE_HELM_CHART::nessie-helm-${RELEASE_VERSION}.tgz
 
+      # Publish Nessie Helm chart to Helm Repo
```
Comment (snazy) [inline .github/workflows/release-publish.yml:234]:
```suggestion
    # Publish Nessie Helm chart to Helm Repo
```
Indent should be fixed (for IDEs)

Comment (nastra) [reply .github/workflows/release-publish.yml:234]: fixed

**Review by snazy — APPROVED** [review]

**Pull request merged**